platform: add radxa rock 5b+ support (#16)
RadxaYuntian/mraa: docs/radxa_rock_5b.md

> Radxa ROCK 5B/5B+ {#_Radxa}

Reviewer: ```suggestion
Radxa ROCK 5B & 5B+ {#_Radxa}
```

===========================

> Radxa ROCK 5B/5B+ is a Rockchip RK3588 based SBC(Single Board Computer) by Radxa. It can run Android or Linux. Radxa ROCK 5B/5B+ features a four core ARM processor, 64bit dual channel 5500Mb/s LPDDR5, HDMI up to 8K60fps, MIPI DSI, MIPI CSI, 3.5mm combo audio jack, Wi-Fi 6, Bluetooth 5.0, USB, GbE LAN, and 40-pin color expansion header. Radxa ROCK 5B/5B+ is powered by the USB Type-C port, and supports 5V input only. The recommended power adapter is 5V/3A without SSD, or 5V/5A with SSD.

Reviewer: ```suggestion
Radxa ROCK 5B & 5B+ are Rockchip RK3588 based SBCs (Single Board Computer) made by Radxa. It can run Android or Linux. Radxa ROCK 5B & 5B+ feature a eight core ARM processor, 64bit dual channel 5500Mb/s LPDDR5, HDMI up to 8K60fps, MIPI DSI, MIPI CSI, 3.5mm combo audio jack, USB, ethernet, and 40-pin GPIO header. Radxa ROCK 5B & 5B+ are powered by the USB Type-C port, with 5B+ having 2 power input ports. The recommended power adapter is 5V/3A without SSD, or 5V/5A with SSD.
```

Pin Mapping
-----------

> Radxa ROCK 5B/5B+ has a 40-pin expansion header. Each pin is distinguished by the color.

Reviewer: ```suggestion
Radxa ROCK 5B & 5B+ has a 40-pin expansion header. Pins are color coded.
```

> |  Function5|    Function4|    Function3|    Function2|  Function1|  PIN  |  PIN  | Function1|    Function2|   Function3|   Function4|

Reviewer: 5B+下面不是至少两个引脚都和5B不同么？

- [Product Info](https://docs.radxa.com/en/rock5/rock5b)
- [Forums](https://forum.radxa.com/c/rock5)

> - [Github](https://github.com/radxa)

Reviewer: 这改了啥？
